src/main/java/uk/gov/hmcts/reform/civil/model/citizenui/DashboardClaimantClaimStatusFactory.java

```diff
@@ -0,0 +1,16 @@
+package uk.gov.hmcts.reform.civil.model.citizenui;
```

Comment:
There is no need for this class.  You can use the existing DashboardClaimStatusFactory because it is doing the same thing.  You will just supply it with the alternative matcher.

Reply:
DashboardClaimStatusFactory can stay exactly the same as it was.  There is no need to have two different ClaimStatusFactories, because they are both doing the same thing.  It is only the matcher that we need to have two of.